packages/flutter/lib/animation.dart

```diff
@@ -2,6 +2,8 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+/// @docImport 'package:flutter/material.dart';
+///
 /// The Flutter animation system.
 ///
 /// To use, import `package:flutter/animation.dart`.
```

Comment:
@srawlins does it make sense to put these at the top of the already present doc comment on the library statement?

Reply:
Hm, I guess this doesn't quite work yet as dartdoc puts that string into the output (see screenshot below). Does this need a fix in dartdoc?

![Screenshot 2024-06-28 at 2 22 22 PM](https://github.com/flutter/flutter/assets/1227763/d79d52bb-f8e1-4720-8d66-4545013e0049)

Reply:
It also shows up in the libraries overview:

![Screenshot 2024-06-28 at 2 24 23 PM](https://github.com/flutter/flutter/assets/1227763/8eb5bfcb-e388-4fd1-ad8d-e0e8ee18e179)

Reply:
I fixed this yesterday and it will be released with dartdoc 8.0.10: https://github.com/dart-lang/dartdoc/pull/3806/files

Reply:
Awesome. I verified locally that it works with this change. https://github.com/flutter/flutter/pull/151107 bumps the dartdoc version.

Reply:
Excellent!

Reply:
> I would probably lean towards putting them at the bottom, as it would be tedious to read the library doc comments in source if they started with a handful of doc-imports.

That's a good point, I'll move these to the bottom. I low-key want a lint for this :P